platform-includes/metrics/configure/javascript.astro.mdx

```diff
@@ -6,12 +6,20 @@ Metrics work out of the box by calling `Sentry.init` no further setup is require
 // sentry.client.config.js
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
+  // Only needed for SDK versions < 8.0.0
+  // integrations: [
+  //   Sentry.metrics.metricsAggregatorIntegration(),
```

Comment:
Hmm, where does this come from? We don't really do this for any other thing, we always show up-to-date docs as far as I can tell? 🤔

Reply:
Because of metrics GA happening so close to v8, we had some confusion about this and people had troubles intrumenting custom metrics. Daniel gave the green light to have a short-term callout for this in docs.

Reply:
Users that still use v7 were confused during the onboarding process.
We discussed it with @HazAT as a temporary measure for metrics GA in the planning today.

Reply:
alright, all good then!